### PR TITLE
Singleton init checks

### PR DESCRIPTION
**Pull request by jamesstout**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Ensures the query controler is init on main

Does this close any currently open issues?
------------------------------------------
…

Any relevant logs, error output, etc?
-------------------------------------

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Devices/Simulators:** iMac19,1

**macOS Version:**: 10.15.7 (19H2)

**Sequel-Ace Version:** arc_updates latest

### CODE REVIEW

File: Resources/License.rtf [hunk 1]
```diff
@@ -1,1148 +1,427 @@
-{\rtf1\ansi\ansicpg1252\cocoartf2576
-\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica-Bold;\f1\fswiss\fcharset0 Helvetica;\f2\fmodern\fcharset0 Courier;
+{\rtf1\ansi\deff0{\fonttbl{\f0 \fswiss Helvetica;}{\f1 Courier;}}
```
Comment (Kaspik) [inline Resources/License.rtf:1]: Changed again? 😄

File: Source/Controllers/SubviewControllers/SPQueryController.m [hunk 2]
```diff
@@ -74,7 +74,14 @@ + (SPQueryController *)sharedQueryController
 {
 	@synchronized(self) {
 		if (sharedQueryController == nil) {
-			sharedQueryController = [[super allocWithZone:NULL] init];
+			if (![NSThread isMainThread]){
+				SPMainQSync(^{
+					sharedQueryController = [[super allocWithZone:NULL] init];
```
Comment (Kaspik) [inline Source/Controllers/SubviewControllers/SPQueryController.m:79]: not sure about that... why can't we just `[[SPQueryController alloc] init]`?

File: Resources/License.rtf (hunk 1 above)
Comment (jamesstout) [reply Resources/License.rtf:1]: the RTF was all screwed up, no changes I don't think.

File: Source/Controllers/SubviewControllers/SPQueryController.m (hunk 2 above)
Comment (jamesstout) [reply Source/Controllers/SubviewControllers/SPQueryController.m:79]: goes into some weird loop..

Comment (jamesstout) [reply Source/Controllers/SubviewControllers/SPQueryController.m:79]: we can remove allocWithZone:(NSZone *)zone and then there's no loop.

**Review by Kaspik — APPROVED** [review]

**Pull request merged**